src/Platform/Microsoft.Testing.Platform/Capabilities/TestFramework/IBannerMessageOwnerCapability.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Testing.Platform.Capabilities.TestFramework;
+
+public interface IBannerMessageOwnerCapability : ITestFrameworkCapability
+{
+    string GetBannerMessage();
```

Comment:
Async could be good...some scanning of files here could be done by the adapters, also I think that when we register we offer the service provider so in case the informations could be taken from there when we decide how to expose these informations, otherwise today means have a simple placehoder IContextInformation or something similar

Reply:
I think that we should implement it for MSTest so we can shape the most interesting information for it.